security/secret-protection/README.adoc

```diff
@@ -5,161 +5,8 @@
-----
-
-The configuration file `connect-avro-distributed.properties` does not change because it's just a pointer to the secrets file.
-However the secrets file has a new value for the encrypted value for this file/parameter pair:
-
-[source,bash]
-----
-$ cat /path/to/secrets.txt
-...
-connect-avro-distributed.properties/config.storage.topic = ENC[AES/CBC/PKCS5Padding,data:CblF3k1ieNkFJzlJ51qAAA==,iv:dnZwEAm1rpLyf48pvy/T6w==,type:str]
-----
-
-=== Trust but verify
-
-That's cool, but does it work?
-Try it out yourself.
-Run Kafka and start the modified connect worker with the encrypted value of `config.storage.topic=newTopicName`
-
-[source,bash]
-----
-# Start ZooKeeper and a Kafka broker
-$ confluent local start kafka
-
-# Run the modified connect worker
-$ connect-distributed connect-avro-distributed.properties > connect.stdout 2>&1 &
-
-# List the topics
-$ kafka-topics --bootstrap-server localhost:9092 --list
-__confluent.support.metrics
-__consumer_offsets
-_confluent-metrics
-connect-offsets
-connect-statuses
-newTopicName   <<<<<<<
-----
-
-=== Going to production
-
-So far we have covered how to create the master encryption key and encrypt secrets in the configuration files.
-We recommend that you operationalize this workflow by augmenting your orchestration tooling to distribute everything that you need for secret protection to work to the destination hosts.
-These hosts may include Kafka brokers, Connect workers, Confluent Schema Registry instances, ksqlDB servers, Confluent Control Center, etc., any service using password encryption.
-The CLI is flexible to accommodate whatever secret distribution model you prefer: you can either do the secret generation and configuration modification on each destination host directly, or do it all a single host and then distribute the encrypted secrets to the destination hosts.
-There are four required tasks:
-
-. Export the master encryption key into the environment on every host that will have a configuration file with secret protection.
-. Distribute the secrets file: copy the secrets file `/path/to/secrets.txt` from the local host on which you have been working to `/path/to/secrets-remote.txt` on the destination hosts.
-. Propagate the necessary configuration file changes: update the configuration file on all hosts so that the configuration parameter now has the tuple for secrets.
-. Restart the services if they were already running.
-
-You may also have a requirement to rotate the master encryption key or data encryption key on a regular basis.
-You can do either of these with the CLI, and the example below is for rotating just the data encryption key.
-
-[source,bash]
-----
-$ confluent secret file rotate --data-key --local-secrets-file /path/to/secrets.txt --passphrase @/path/to/passphrase.txt
-----
+You can find the documentation and instructions for running this Secret
+Protection example at
```

Comment:
@tikimims should or shouldn't be a new line here?

Reply:
Oh I wrapped the text here. I'll unwrap